rpcs3/rpcs3qt/emu_settings.cpp

```diff
@@ -307,6 +307,86 @@ void emu_settings::EnhanceCheckBox(QCheckBox* checkbox, emu_settings_type type)
 	});
 }
 
+void emu_settings::EnhanceDateTimeEdit(QDateTimeEdit* date_time_edit, emu_settings_type type, const QString& format, bool use_calendar, bool as_offset_from_now)
+{
+	if (!date_time_edit)
+	{
+		cfg_log.fatal("EnhanceDateTimeEdit '%s' was used with an invalid object", cfg_adapter::get_setting_name(type));
+		return;
+	}
+
+	date_time_edit->setDisplayFormat(format);
+	date_time_edit->setCalendarPopup(use_calendar);
+
+	if (as_offset_from_now)
+	{
+		bool ok_def = false;
+		const s64 def = qstr(GetSettingDefault(type)).toLongLong(&ok_def);
+		if (!ok_def)
+		{
+			cfg_log.fatal("EnhanceDateTimeEdit '%s' was used with an invalid emu_settings_type", cfg_adapter::get_setting_name(type));
+			return;
+		}
+
+		bool ok_sel = false;
+		s64 val = qstr(GetSetting(type)).toLongLong(&ok_sel);
+		if (!ok_sel)
+		{
+			cfg_log.error("EnhanceDateTimeEdit '%s' tried to set an invalid value: %d. Setting to default: %d", cfg_adapter::get_setting_name(type), val, def);
+			val = def;
+			m_broken_types.insert(type);
+			SetSetting(type, std::to_string(def));
+		}
+
+		// get the date_time value with the offset applied, and set it to the control
+		const QDateTime date_time = QDateTime::currentDateTime().addSecs(val);
+		date_time_edit->setDateTime(date_time);
+	}
+	else
+	{
+		auto str = qstr(GetSettingDefault(type));
+		const auto def = QDateTime::fromString(str, Qt::ISODate);
+		const bool ok_def = def.isValid();
+		if (!ok_def)
+		{
+			cfg_log.fatal("EnhanceDateTimeEdit '%s' was used with an invalid emu_settings_type", cfg_adapter::get_setting_name(type));
+			return;
+		}
+
+		str = qstr(GetSetting(type));
+		auto val = QDateTime::fromString(str, Qt::ISODate);
+		const bool ok_sel = val.isValid();
+		if (!ok_sel)
+		{
+			cfg_log.error("EnhanceDateTimeEdit '%s' tried to set an invalid value: %s. Setting to default: %s", \
```

Comment:
Why the \

Reply:
VS Intellisense took too long to correct the red squiggles, and my mind switched languages... but it's still valid C++...
I do find that it adds a little to the readability (but that might be uniquely for me).
But it is superfluous, so it's gone.